src/lib/dnssd/minimal_mdns/ResponseSender.h

```diff
@@ -88,8 +105,13 @@ class ResponseSendingState
 class ResponseSender : public ResponderDelegate
 {
 public:
-    // TODO(cecille): Template this and set appropriately. Please see issue #8000.
+    // Generally this is 2 + number of operational networks required, based on
+    // the current implementation of Advertiser_ImplMinimalMdns.cpp:
+    //    - 1 for commissionable advertising
+    //    - 1 for commissioner responder
+    //    - extra for every operational advertisement
     static constexpr size_t kMaxQueryResponders = 7;
```

Comment:
Should this also be using CHIP_CONFIG_MAX_FABRICS?  That said, this seems to be test-only now.  Should this move into the test?

Reply:
updated code and test (test now loops in an add and checks that only added once).